frontend/src/components/Card.tsx

```
@@ -44,12 +44,13 @@ const Card = ({
 
   return (
     <div className="mb-2 mt-4 flex w-full flex-col items-start rounded-md border border-border bg-white pb-4 pl-4 dark:bg-[#212529] md:max-w-6xl">
+      {/* Header Section */}
       <div className="mt-2 flex w-full flex-col items-start gap-4 pt-2 sm:flex-col sm:gap-4 md:pt-0">
         <div className="flex items-center gap-3">
           {/* Display project level badge (if available) */}
           {level && (
             <Tooltip
-              closeDelay={100}
+              id={`level-tooltip-${title}`}  // ✅ FIX: Unique tooltip ID
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)